state/mysql/mysql.go

```diff
@@ -759,14 +773,24 @@ func (m *MySQL) BulkGet(parentCtx context.Context, req []state.GetRequest, _ sta
 	return res[:n], nil
 }
 
-func readRow(row interface{ Scan(dest ...any) error }) (key string, value []byte, etagP *string, err error) {
+func readRow(row interface{ Scan(dest ...any) error }) (key string, value []byte, etagP *string, expireTime *time.Time, err error) {
 	var (
 		etag     string
 		isBinary bool
+		expire   sql.NullString
```

Comment:
Thoughts on making this just a string? Should be more efficient. If the value is NULL in the database, I presume this would be an empty string, and then we can skip the parsing.

Reply:
Unfortunately, this is not supported:

```
        	            	error getting state: rpc error: code = Internal desc = fail to get stable-certification-key1 from state store mysql: sql: Scan error on column index 4, name "expiredate": converting NULL to string is unsupported
```

Reply:
Use `SELECT IFNULL(expiredate, "")`